entrypoint

```diff
@@ -51,6 +51,26 @@ if [ ! -e "\$PREF_FILE" ]; then
     source gen-config.sh
 fi
 
+# Changes the log verbosity
+if [ "$LOGVERBOSE" = "1" ] || [ -z "$LOGVERBOSE" ]; then
```

Comment:
I've been thinking about this and I'm not sure how to handle "truthy" values. Whether that is any non-empty value or some specific truthy values like `true`, `1` etc. Either way as is I don't think it'll work.

Perhaps something like this

```
bool() {
    case $(echo "$1" | tr 'A-Z' 'a-z') in
        1|t|true) return 0;;
        *) return 1;;
    esac;
}

if bool "$LOG_VERBOSE"; then
    ...
```

Reply:
I am just using the Plex built-in options, which are `0` or `1` for `LogVerbose`. I use `[ -z "$LOGVERBOSE" ]` as a default value (keep it as it is: `1`)